src/DistributedFFT.chpl

```diff
@@ -2,6 +2,7 @@
 prototype module DistributedFFT {
 
   use BlockDist;
+  use ChapelLocks;
```

Comment:
Out of curiosity, was this something I just forgot? I know I was using a local spinlock somewhere, but I don't see any related changes below....

Reply:
Upstream change -- https://github.com/chapel-lang/chapel/pull/13822 (ChapelLocks wasn't supposed to be included by default.) I had to make this change in order to run with 1.20, but forgot to contribute this earlier.